azure-pipelines.yml

```diff
@@ -14,113 +14,115 @@ pr:
+  parameters:
+    jobName: Build_Windows_Release
+    testArtifactName: Test_Windows_Release
+    configuration: Release
+
+- template: eng/pipelines/test-windows-job.yml
+  parameters:
+    testRunName: 'Test Windows Desktop Debug 32'
+    jobName: Test_Windows_Desktop_Debug_32
+    buildJobName: Build_Windows_Debug
+    testArtifactName: Test_Windows_Debug
+    configuration: Debug
+    testArguments: -testDesktop -test32
+
+- template: eng/pipelines/test-windows-job.yml
+  parameters:
+    testRunName: 'Test Windows Desktop Spanish Debug 32'
+    jobName: Test_Windows_Desktop_Spanish_Debug_32
+    buildJobName: Build_Windows_Debug
+    testArtifactName: Test_Windows_Debug
+    configuration: Debug
+    testArguments: -testDesktop -test32
+    queueName: 'BuildPool.Windows.10.Amd64.ES.VS2017.Open'
+
+- template: eng/pipelines/test-windows-job.yml
+  parameters:
+    testRunName: 'Test Windows Desktop Debug 64'
+    jobName: Test_Windows_Desktop_Debug_64
+    buildJobName: Build_Windows_Debug
+    testArtifactName: Test_Windows_Debug
+    configuration: Debug
+    testArguments: -testDesktop -test64
+
+- template: eng/pipelines/test-windows-job.yml
+  parameters:
+    testRunName: 'Test Windows CoreClr Debug'
+    jobName: Test_Windows_CoreClr_Debug
+    buildJobName: Build_Windows_Debug
+    testArtifactName: Test_Windows_Debug
+    configuration: Debug
+    testArguments: -testCoreClr
+
+- template: eng/pipelines/test-windows-job.yml
+  parameters:
+    testRunName: 'Test Windows Desktop Release 32'
+    jobName: Test_Windows_Desktop_Release_32
+    buildJobName: Build_Windows_Release
+    testArtifactName: Test_Windows_Release
+    configuration: Release
+    testArguments: -testDesktop -test32
+
+- template: eng/pipelines/test-windows-job.yml
+  parameters:
+    testRunName: 'Test Windows Desktop Release 64'
+    jobName: Test_Windows_Desktop_Release_64
+    buildJobName: Build_Windows_Release
+    testArtifactName: Test_Windows_Release
+    configuration: Release
+    testArguments: -testDesktop -test64
+
+- template: eng/pipelines/test-windows-job.yml
+  parameters:
+    testRunName: 'Test Windows CoreClr Release'
+    jobName: Test_Windows_CoreClr_Release
+    buildJobName: Build_Windows_Release
+    testArtifactName: Test_Windows_Release
+    configuration: Release
+    testArguments: -testCoreClr
+
+# Unix Build and Test Jobs 
+- template: eng/pipelines/build-unix-job.yml
+  parameters:
+    jobName: Build_Unix_Debug
+    testArtifactName: Test_Unix_Debug
+    configuration: Debug
+    vmImageName: 'ubuntu-16.04'
+
+- template: eng/pipelines/test-unix-job.yml
+  parameters:
+    testRunName: 'Test Linux Debug'
+    jobName: Test_Linux_Debug
+    buildJobName: Build_Unix_Debug
+    testArtifactName: Test_Unix_Debug
+    configuration: Debug
+    testArguments: --testCoreClr
+    queueName: 'BuildPool.Ubuntu.1604.amd64.Open'
+
+- template: eng/pipelines/test-unix-job.yml
+  parameters:
+    testRunName: 'Test OSX Debug'
```

Comment:
`Test macOS Debug`? (and in the jobName below)

Reply:
The other repos seem to lean towards the OSX terminology. Is there a reason you believe macOS is more appropriate here?